src/ethereum_spec_tools/diff.py

```diff
@@ -40,30 +40,23 @@ def find_pickles(path: str, fork: Hardfork) -> Iterator[str]:
 
 
 def meaningful_diffs(
-    pub: rstdiff.Publisher3Args,
     old: Any,
     new: Any,
+    trivial_changes: List[Tuple[str, str, re.RegexFlag]],
 ) -> bool:
     """Find if there are meaningful differences between the docs"""
-    realDebug = pub.settings.debug
-    pub.settings.debug = pub.settings.dump_rstdiff
-    reporter = new_reporter("RSTDIFF", pub.settings)
-    pub.settings.debug = realDebug
-    dispatcher = rstdiff.DocutilsDispatcher(reporter)
-    opcodes = rstdiff.doDiff(dispatcher, old, new)
-
-    opcode_counter = rstdiff.OpcodeCounter(opcodes)
-    opcode_counter.count()
-
-    return bool(
-        opcode_counter.replace
-        or opcode_counter.delete
-        or opcode_counter.insert
-    )
+    old = re.sub(r"[\n\t\s]", "", str(old))
+    new = re.sub(r"[\n\t\s]", "", str(new))
```

Comment:
Two minor suggestions here: 
  - Compile the regex
  - Convert whitespace to a space, instead of the empty string. `Foo Bar` to `FooBar` is a pretty meaningful change IMO.

```suggestion
    pattern = re.compile(r"[\n\t\s]")  # Can probably move to a package-level constant.
    old = pattern.sub(" ", str(old))
    new = pattern.sub(" ", str(new))
```

Reply:
Updated.